### ISSUE TITLE
GCC Support

### ISSUE THREAD

**Issue by hlord2000** [jj8431/cpp_nRF5_BLE_SDK#1]
Hello! I love this project, very useful for what I'm working on now. Would you be willing to accept a PR for GCC support with the supplied makefiles used in Nordic's examples? Thanks.

**Comment by timj-code** [comment]
Yes that would be nice, it will let more people to use it easily.
